docs/diagrams/CountryNotePanelClientViewActDiag.puml

```diff
@@ -0,0 +1,27 @@
+@startuml
+start
+:User inputs a command;
+
+'Since the beta syntax does not support placing the condition outside the
+'diamond we place it as the true branch instead.
+
+:LogicManager executes the user command
+and returns an instance of CommandResult;
+
+if () then ([commandResult.shouldDisplayClient()])
+    :Remove all child components in
+    the Display Panel Container;
+    :Add the Container of the
+    Client View UI as a child component
+    of the Display Panel Container;
```

Comment:
Is it possible to have an `else` after this `if` so that the diagram looks better? Like the right arrow coming from the first diamond should have an `[else]`

Reply:
I changed to using if/else instead of if/else if as it seems like plantuml does not support the feature you described.